docs/operator-guide/troubleshooting/application-not-built.md

```diff
@@ -1,32 +1,24 @@
-# Codebase Build Process is Failed
+# Codebase Review Process is Failed
```

Comment:
Review Pipeline is Failed because of SonarQube

Reply:
let's generalize this case, we can remove sonarqube or leave it as example. But let's make it more general, and it should be "Review Pipelines Fails"